cluster/examples/kubernetes/ceph/monitoring/grafana.yaml

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
How about creating an integration test that will install all these monitoring yamls? Even if we don't test the full end to end gathering of ceph metrics, at least we can verify basic install?